opm/parser/eclipse/EclipseState/Grid/GridProperties.hpp

```diff
@@ -82,33 +95,12 @@ namespace Opm {
             if (!hasKeyword(keyword))
                 addAutoGeneratedKeyword_(keyword);
 
-            return *m_properties.at( keyword );
-        }
-
-        GridProperty<T>& getKeyword(const std::string& keyword) {
-            if (!hasKeyword(keyword))
-                addAutoGeneratedKeyword_(keyword);
-
-            return *m_properties.at( keyword );
-        }
-
-        const GridProperty<T>& getKeyword(size_t index) const {
-            if (index < size())
-                return *m_property_list[index];
-            else
-                throw std::invalid_argument("Invalid index");
-        }
-
-        GridProperty<T>& getKeyword(size_t index)  {
-            if (index < size())
-                return *m_property_list[index];
-            else
-                throw std::invalid_argument( "Invalid index" );
+            GridProperty<T>& property = (*m_properties.at( keyword ));//const_cast<const GridProperty<T>&> (*m_properties.at( keyword ));
```

Comment:
Looks like a ghetto const cast to obtain a mutable reference from a const context.